v_3_1_1/types.go

```diff
@@ -66,11 +66,11 @@ type HyperkubeApiserver struct {
 	//
 	// azure-operator sets that to 0.0.0.0. Other operators leave it empty.
 	BindAddress string
-	Docker      HyperkubeDocker
```

Comment:
This one become useless for ApiServe rand ControllerManager as they don't need docker extra arguments, but instead - extra mounts